src/app/components/UpgradeModal.js

```diff
@@ -14,6 +14,7 @@ const UpgradeModal = ({ onConfirm, onClose }) => {
             }}
             onClose={onClose}
```

Comment:
`onClose` was not enough? 
Ping @mmso

Reply:
`UpgradeModal` was called in two places, in one (from `ContactModal`) it was passed an `onClose`, but not on the other (from the sidebar, when clicking for instance 'Groups' with a free user)

Reply:
Rest needs to be passed